rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -128,6 +133,7 @@ public BookKeeper(Configuration conf)
             throw new TException(e);
         }
         endBlock = setCorrectEndBlock(endBlock, fileLength, remotePath);
+        log.info(" Start Block : " + startBlock + " End Block : " + endBlock);
```

Comment:
there will too many of these requests and logging so much data might cause problems. If not perf issues, it might cause undue pressure of logging system that important logs might get delayed from being flushed

Reply:
Will make it debug log.